assets/react/components/home.js

```diff
@@ -31,7 +35,9 @@ module.exports = React.createClass({
 		// }
 	},
 	componentDidMount: function(){
+		document.getElementsByClassName('cards-container')[0].style.visibility = "hidden";
```

Comment:
can we do this with state. Add a class with hidden css and then remove that class with updating the state of the component when masonary is initialized.